modules/core/src/main/java/org/apache/ignite/internal/processors/cache/CacheMetricsImpl.java

```diff
@@ -61,6 +62,10 @@
      */
     public static final String CACHE_METRICS = "cache";
 
+    /** Histogram buckets for duration get, put, remove operations in nanoseconds. */
+    public static final long[] HISTOGRAM_BUCKETS = new long[] {1_000_000, 10_000_000, 100_000_000, 250_000_000,
```

Comment:
Can you, please, use `TimeUnit#convert` here to clarify the numbers for the reader.

Reply:
Done.